apps/src/code-studio/components/progress/SendLessonDialog.jsx

```diff
@@ -0,0 +1,97 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import Button from '@cdo/apps/templates/Button';
+import DialogFooter from '@cdo/apps/templates/teacherDashboard/DialogFooter';
+
+export default class SendLessonDialog extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool.isRequired,
+    handleClose: PropTypes.func.isRequired,
+    showGoogleButton: PropTypes.bool
+  };
+
+  render() {
+    return (
+      <BaseDialog
+        isOpen={this.props.isOpen}
+        handleClose={this.props.handleClose}
+        style={styles.dialog}
+        useUpdatedStyles
+      >
+        <h2>{i18n.sendLessonTitle()}</h2>
+        <div style={styles.detailsLine}>
+          {i18n.sendLessonDetails()}{' '}
+          <a
+            target="_blank"
+            href="https://support.code.org/" // TODO: Update this!
+          >
+            {i18n.learnMore()}
+          </a>
+        </div>
+        {this.renderCopyToClipboardRow()}
+        {this.props.showGoogleButton && this.renderShareToGoogleRow()}
```

Comment:
Not sure if you're at the point where you want to start writing tests yet, but if you are, this would be a good case to unit test.
Example: https://github.com/code-dot-org/code-dot-org/blob/325379a7b21bbf4c7fdc3aba4830f651f7b05dcd/apps/test/unit/code-studio/components/pairing/StudentSelectorTest.js#L16-L23

Reply:
Thanks for the pointer -- I will add this test to check for the google button when I put in the real implementation of the button.